cloudpickle/cloudpickle.py

```diff
@@ -581,6 +581,11 @@ def save_dynamic_class(self, obj):
         if isinstance(__dict__, property):
             type_kwargs['__dict__'] = __dict__
 
+        if sys.version_info < (3, 7):
```

Comment:
The new tests introduced in this PR cover the case where `sys.version_info >= (3, 7)`. We should introduce a new test precisely for Python 3.6 no?

Reply:
No, the new tests cover precisely the case where `Python < 3.7` :)